triton/tut/parallel-shared.rst

```diff
@@ -0,0 +1,257 @@
+===========================================================
+Shared memory parallelism: multithreading & multiprocessing
+===========================================================
+
+
+.. admonition:: Abstract
+
+   * Verify that your program can utilize multiple CPUs.
+
+   * Use ``--cpus-per-task=C`` to reserve multiple CPUs for your job.
+
+   * If you use ``srun`` to launch your program in your sbatch-script and
+     want your program to utilize all of the allocated CPUs, run
+     ``export SRUN_CPUS_PER_TASK=$SLURM_CPUS_PER_TASK`` in your script
+     before calling ``srun``.
```

Comment:
```suggestion
     before calling ``srun`` (new in mid-2022).
```

Reply:
I don't think setting this time information is relevant for anyone reading the document (if they are new users / they look for up to date information).